WORKSPACE

```diff
@@ -47,6 +47,11 @@ http_archive(
     strip_prefix = "rules_apple-0.13.0",
     urls = ["https://github.com/bazelbuild/rules_apple/archive/0.13.0.tar.gz"],
 )
+http_archive(
```

Comment:
This should not be needed. Swift rules are pulled in via Apple rules:

https://github.com/bazelbuild/rules_apple/blob/master/apple/repositories.bzl#L115

Reference: #25735.